src/services/notifier/hooks/plans.hook.ts

```diff
@@ -61,19 +54,15 @@ export default {
         } = query
 
         if (channels?.length) {
-          paramsSeq.include = [
-            ...paramsSeq.include,
-            {
-              model: NotifierChannelModel,
-              as: 'channels',
-              required: true,
-              attributes: ['name'],
-              where: {
-                name: channels
+          paramsSeq.where = {
+            ...paramsSeq.where ?? {},
+            channels: {
+              [Op.contains]: {
+                type: channels
               }
             }
-          ]
-        }
+          }
+        } // TODO: needs testing
```

Comment:
I think it could be better to remove it then or update the comment to "not used yet and not tested"